### PR TITLE
Add precommit and contributing docs

### PR DESCRIPTION
**Pull request by NiallRees**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,3 +53,81 @@ USE SCHEMA your_default_dbt_schema
...
+
+3. Install pre-commit
+```bash
+pipx install pre-commit
+```
+
+4. Configure your profile (follow the prompts)
+```
+dbt init
+```
```
Comment (ian-whitestone) [inline README.md:80]: this creates a new `dbt-project.yml` right? do we want that?

Comment (NiallRees) [reply README.md:80]: When run inside an existing project it just configures `~/.dbt/profiles.yml`.

Comment (ian-whitestone) [reply README.md:80]: ah amazing!

File: README.md [hunk 2]
```diff
@@ -53,3 +53,81 @@ USE SCHEMA your_default_dbt_schema
...
+Install pipx:
+```bash
+pip install pipx
+pipx ensurepath
+```
+
+Install tox:
+```bash
+pipx install tox
+```
```
Comment (ian-whitestone) [inline README.md:109]: @NiallRees this a repeat of above. need it here as well?

**Pull request merged**